indexer/services/ender/__tests__/scripts/scripts.test.ts

```diff
@@ -236,6 +315,174 @@ describe('SQL Function Tests', () => {
     result = await getSingleRawQueryResultRow(`SELECT dydx_uuid_from_subaccount_id_parts('${subaccountId.owner}', '${subaccountId.number}') AS result`);
     expect(result).toEqual(SubaccountTable.subaccountIdToUuid(subaccountId));
   });
+
+  it.each([
+    {
+      event: { transactionIndex: 123 },
+      expectedResult: 123,
+    },
+    {
+      event: { blockEvent: IndexerTendermintEvent_BlockEvent.BLOCK_EVENT_BEGIN_BLOCK },
+      expectedResult: -2,
+    },
+    {
+      event: { blockEvent: IndexerTendermintEvent_BlockEvent.BLOCK_EVENT_END_BLOCK },
+      expectedResult: -1,
+    },
+    {
+      event: { blockEvent: '3' },
+      expectedError: 'Received V4 event with invalid block event type: 3',
+    },
+    {
+      event: {},
+      expectedError: 'Either transactionIndex or blockEvent must be defined in IndexerTendermintEvent',
+    },
+  ])('dydx_tendermint_event_to_transaction_index should return the expected result', async (
+    { event, expectedResult, expectedError },
+  ) => {
+    try {
+      const result = await getSingleRawQueryResultRow(
+        `SELECT dydx_tendermint_event_to_transaction_index('${JSON.stringify(event)}') AS result`);
+      if (expectedError) {
+        throw new Error('Expected an error but got a result.');
+      }
+      expect(result).toEqual(expectedResult);
+    } catch (error) {
+      if (!expectedError) {
+        throw error;
+      }
+      expect(error.message).toContain(expectedError);
+    }
+  });
+
+  it.each([
+    {
+      blockHeight: '123456',
+      transactionIndex: 123,
+    },
+  ])('dydx_uuid_from_transaction_parts (%s)', async (transactionParts) => {
+    const result = await getSingleRawQueryResultRow(
+      `SELECT dydx_uuid_from_transaction_parts('${transactionParts.blockHeight}', '${transactionParts.transactionIndex}') AS result`);
+    expect(result).toEqual(
+      TransactionTable.uuid(transactionParts.blockHeight, transactionParts.transactionIndex),
+    );
+  });
+
+  it('dydx_create_transaction.sql should insert a transaction and return correct jsonb', async () => {
```

Comment:
nit: for organization either move tests for each sql function to a separate file or nest it inside of a `describe` block.

Reply:
I'm following the existing organization - there is a test per sql function. Do you want me to move all the new tests into a separate `describe` block?